src/test/java/GeneralService/CsvFileReaderTest.java

```diff
@@ -0,0 +1,22 @@
+package GeneralService;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import utils.filereaders.CsvFileReader;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class CsvFileReaderTest {
+    private final CsvFileReader csvFileReader = new CsvFileReader();
+
+    @Test
+    void correctTest() {
+        List<String> list = csvFileReader.readFile(new File(".\\src\\main\\resources\\test.csv"));
```

Comment:
instead of actually using a real file, create a temp one with the data in the code, and then delete the test file after the test is complete